src/inject.js

```diff
@@ -44,6 +45,24 @@ export default function injectDynamicImport(acorn) {
         return nextMethod.call(this, refDestructuringErrors);
       }
     ));
+
+    instance.extend('parseExpression', nextMethod => (
+      function parseExpression() {
+        const node = nextMethod.call(this);
+        if (node.type === 'CallExpression' && node.callee.type === 'Import') {
+          if (node.arguments.length !== 1) {
+            this.raise(node.start, 'import() requires exactly one argument');
+          }
+
+          const importArg = node.arguments[0];
+          if (importArg && importArg.type === 'SpreadElement') {
+            this.raise(importArg.start, '... is not allowed in import()');
+          }
+        }
+
+        return node;
+      }
+    ));
```

Comment:
Oh awesome, didn't know you ended up replicating this in esm. Would love to see if we can fix this implementation so that you don't have to mirror it there (unless you want to have it mirrored there).

That's probably a better approach. I think this causes it to raise at the start of the call expression based on the generated `expected.json` in this PR.